Quest-3/Part-1/main.py

```diff
@@ -0,0 +1,69 @@
+def main(fileName):
+    lines = readFile(fileName)
+
+    schemEngine = [list(line.replace("\n", "")) for line in lines]
+    symbols = set(char for line in schemEngine for char in line if char != '.' and not char.isdigit())
```

Comment:
this is something really weird. I can't parse it. But thankfully it's probably not needed. You could have a set of of coords of symbols, not symbols themselves.

Reply:
reminding you of that. you don't need `symbols` at all. Just imagine your field is very, very big. You just had an additional scan just to count them. This is wasteful.